src/Stratis.Bitcoin/Controllers/Models/StatusModel.cs

```diff
@@ -27,9 +27,15 @@ public StatusModel()
         /// <summary>The network the current node is running on.</summary>
         public string Network { get; set; }
 
+        /// <summary> The processId of the node.</summary>
+        public int PID { get; set; }
+
         /// <summary>The height of the consensus.</summary>
         public int ConsensusHeight { get; set; }
 
+        /// <summary> The height of the full blocks downloaded to BlockStore </summary>
```

Comment:
`/// <summary>Height of the last block in the block store.</summary>`

Note:
- no extra space at start or end
- dot at the end
- either use `<see>` if you want to talk about name of specific component (or namespace - i.e. `<see cref="BlockStore"/>` would be for namespace, but you would probably need to use full path as you have no usings in this file -> `<see cref="Stratis.Bitcoin.Features.BlockStore"/>`), or use English description without see